dll/win32/ieframe/lang/cs-CZ.rc

```diff
@@ -60,6 +60,10 @@ STRINGTABLE
 
 STRINGTABLE
 {
+#ifdef __REACTOS__
+    IDS_INTERNET            "Internet"
```

Comment:
If this thing opens iexplore.exe unconditionally, I think this should be named `"Wine Internet Explrer"` to really insist that it's going to open Wine IE and not anything else (and that this program really comes from Wine, and not ReactOS.)

Reply:
In Windows XP/2003, when modern start menu is enabled, there is "Internet" menu item, which starts the web browser assigned by the user - it's not necessarily "Internet Explorer":

![image](https://github.com/reactos/reactos/assets/578406/d05ca406-f53a-49ea-8609-86cd5f14c8f4)